apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -2,17 +2,93 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { StyleSheet, Text } from '@kiwicom/universal-components';
+import {
+  StyleSheet,
+  Card,
+  Text,
+  TextInput,
+  Picker,
+} from '@kiwicom/universal-components';
+import { HeaderWithIcon } from '@kiwicom/margarita-components';
 import { withNavigation } from '@kiwicom/margarita-navigation';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
 type Props = {|
   +detailId: ?string,
 |};
 
-class ResultDetail extends React.Component<Props> {
+type State = {|
+  +email: ?string,
+  +phoneNumber: ?string,
+  +phoneCountryCode: ?string,
+|};
+
+class ResultDetail extends React.Component<Props, State> {
+  state = {
+    email: null,
+    phoneNumber: null,
+    phoneCountryCode: null,
+  };
+
+  handleChangeEmail = email => {
+    this.setState({ email });
+  };
+
+  handleChangePhoneNumber = phoneNumber => {
+    this.setState({ phoneNumber });
+  };
+
+  handleChangePhoneCountryCode = phoneCountryCode => {
+    this.setState({ phoneCountryCode });
+  };
+
   render() {
+    const phoneCountryCodeData = [
+      {
+        label: '+420',
+        value: '+420',
+      },
+      {
+        label: '+421',
+        value: '+421',
+      },
+    ];
+
     return (
       <View style={styles.container}>
+        <Card>
+          <HeaderWithIcon label="Contact details" iconName="contact-email" />
+          <View style={styles.line}>
+            <TextInput
+              onChangeText={this.handleChangeEmail}
+              label="E-mail"
+              type="email"
+              autoCorrect={false}
+            />
+          </View>
+          <View style={[styles.line, styles.row]}>
+            <View style={styles.countryCode}>
+              <Picker
+                selectedValue="+420"
```

Comment:
We should probably use `selectedValue={this.state.phoneCountryCode}` for now. 
And later we can change default `phoneCountryCode` value according to location (but it is long-term nice to have).

Reply:
There is a problem that `selectedValue` could be `null`, which is not allowed in the `Picker`. Actually I think it's a bug in `Picker`, but there should be possibility of not choosing any value. Right now you have `placeholder` in the `Picker` which should be invisible all the time, because there is no way how to not set up the value.

Reply:
Right. We should allow `Picker` selected value to be null. What about like this https://github.com/kiwicom/universal-components/pull/256 ?

Reply:
Perfect, thanks! 

It's merged, so I will change it and it should work in new `universal-components` release